draft-ietf-core-senml.md

```diff
@@ -540,10 +540,12 @@ defined in {{tbl-cbor-labels}}. This table is conclusive, i.e., there is no
 intention to define any additional integer map keys; any extensions will use
 string map keys.
 
-* For JSON Numbers, the CBOR representation can use integers, floating point
-numbers, or decimal fractions (CBOR Tag 4); the common limitations of JSON
-implementations are not relevant for these. For the version number, however,
-only an unsigned integer is allowed.
+* For JSON Numbers, the CBOR representation can use integers, floating
+point numbers, or decimal fractions (CBOR Tag 4); however a
+representation SHOULD be choose such that when it the CBOR value is
+converted back to an IEEE double-precision floating-point value, it
+has exactly the same value as the original Number.  For the version
```

Comment:
Since the IEEE double-precision floating point (binary64) numbers can be directly exchanged in CBOR, all such numbers are representable.  So there is no case where representation of an I-JSON number is impossible.  Using decimal of course has some interesting corner cases, but that is more of a quality of implementation issue — we are making the intention very clear here that binary64 is the "native" number system.